src/Adapter/MSTest.TestAdapter/Execution/TestMethodRunner.cs

```diff
@@ -182,6 +182,7 @@ internal UnitTestResult[] RunTestMethod()
             }
             else
             {
+                _testContext.SetDisplayName(_test.DisplayName);
```

Comment:
Not sure why it needs to be conditional. Any harm doing it unconditionally at the beginning of `RunTestMethod` or even in the constructor [here](https://github.com/microsoft/testfx/blob/15f2bbd7abf8ebae96070006eff2c3bf921b62b8/src/Adapter/MSTest.TestAdapter/Execution/TestMethodRunner.cs#L61-L62)?

Reply:
It's using a different logic for the parameterized tests. I was thinking about setting it always first and then resetting it based on parameterized tests submethods but thought it would be less clear.